app/models/manageiq/providers/cloud_automation_manager/configuration_manager/refresh_parser.rb

```diff
@@ -27,17 +27,18 @@ def configuration_profile_inv_to_hashes(profiles)
         end
       end
 
+      
       def configured_system_inv_to_hashes(configured_systems)
         configured_systems.to_a.collect do |cs|
           {
             :type         => "ManageIQ::Providers::CloudAutomationManager::ConfigurationManager::ConfiguredSystem",
             :manager_ref  => cs["id"].to_s,
             :hostname     => cs["name"],
-            :last_checkin => cs["last_compile"],
-            :build_state  => cs["build"] ? "pending" : nil,
-            :ipaddress    => cs["ip"],
-            :mac_address  => cs["mac"],
-            :ipmi_present => cs["sp_ip"].present?,
+            :last_checkin => cs["created_at"],
+            :build_state  => cs["build"],
+            :ipaddress    => cs["templateId"],
+            :mac_address  => cs["status"],
+            :ipmi_present => cs["templateName"]
```

Comment:
@blancett are these accurate?  IP Address is stored in `templateId`, MAC Address in `status`?

Reply:
yeah i am just cramming in data - not of this makes sense for terraform. If they show up in the table we are good. We will revisit all this after the POC.

Reply:
yes just cramming in data for the ui table - not of it matches - we will have to revisit after the POC.